From: Enrico Tassi <gareuselesinge@debian.org>
Date: Sun, 9 Jun 2013 15:53:14 +0200
Subject: multi-arch-path

===================================================================
---
 src/luaconf.h |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/luaconf.h b/src/luaconf.h
index d55caab..9e2e6f2 100644
--- a/src/luaconf.h
+++ b/src/luaconf.h
@@ -33,7 +33,10 @@
 #define LUA_JDIR	LUA_XROOT "share/luajit-2.0.2/"
 #define LUA_XPATH \
   ";" LUA_XROOT "share/lua/5.1/?.lua;" LUA_XROOT "share/lua/5.1/?/init.lua"
-#define LUA_XCPATH	LUA_XROOT "lib/lua/5.1/?.so;"
+/* This defines DEB_HOST_MULTIARCH */
+#include "luajit5.1-deb-multiarch.h"
+#define LUA_XCPATH	LUA_XROOT "lib/" DEB_HOST_MULTIARCH "/lua/5.1/?.so;" \
+			LUA_XROOT "lib/lua/5.1/?.so;"
 #else
 #define LUA_JDIR	LUA_ROOT "share/luajit-2.0.2/"
 #define LUA_XPATH
